buttoncount.js

```diff
@@ -1,12 +1,8 @@
-function fun(){
+function fun() {
```

Comment:
who so many empty lines are present ??
Use eslint in vs code = Ctrl + shift + I